bower.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/PolymerElements/paper-input",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
+    "polymer": "Polymer/polymer#^1.1.0",
```

Comment:
I don't think this is necessary. `1.1.0` is covered by `^1.0.0` (you can do a `bower cache clean; bower install` to double check